replies.go

```diff
@@ -159,8 +159,7 @@ var replies = []Reply{
 		message: "Impact is generally incompatible with modpacks and support will not be provided if you encounter bugs with them. It's likely your game will just crash on startup.",
 	},
 	{
-		pattern: `good\s*bot`,
-		message: "tnyak yow *nuwzzwes yoww necky wecky*",
+
 	},
```

Comment:
This removes the content of the reply, but doesn't remove the actual reply entry (the `{},` brackets remain)

Reply:
I will fix, I forgot